services/postgres/values.yaml

```diff
@@ -0,0 +1,18 @@
+# Default values for fileserver.
```

Comment:
Needs helm-docs comments. The standard for Helm parameters is studlyCaps, not snake_case. Looks like most of the parameters aren't documented and probably should be.

Reply:
Done (and converted to studlyCaps)